packages/canvas/canvas-text/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "@pixi/canvas-text",
+  "version": "5.0.4",
+  "main": "lib/canvas-text.js",
+  "module": "lib/canvas-text.es.js",
+  "bundle": "dist/canvas-text.js",
+  "bundleNoExports": false,
```

This should be `true` since this doesn't export anything. It's like the mixin* packages.
```suggestion
  "bundleNoExports": true,
```